README.md

```diff
@@ -62,16 +62,20 @@ This will update the collections and pipelines in your environment. New collecti
     ```
    This will copy files from the ./ci/assets directory to a ./ci/release directory and will then push the built images to the specified image registry.
 
-1. [Optional] Create a production Git Release 
+1. [Optional] Create a production Git Release
 
-   If building locally and not using Travis, you will need to create a release on a production GIT repository. 
+   If building locally and not using Travis, you will need to create a release on a production GIT repository.
 
    1. Use these instructions to [create a GIT release manually](https://github.com/kabanero-io/collections/blob/master/create-release.md) using the files in the ./ci/release directory rather than the ./ci/assets directory.
    1. Once all the artifacts are uploaded to the GIT Release, go to the assets inside that release and find the file kabanero-index.yaml and copy its URL for use within your Kabanero instance.
    ```
      e.g. https://github.com/kabanero-io/collections/releases/download/v0.2.0.beta2/kabanero-index.yaml .
    ```
-   
+
+## Github enterprise considerations
+
+When using Github enterprise to store collections you may need an alternative mechanism for hosting your release artefacts due to authentication requirements. See [Hosting your collections using NGINX](https://github.com/kabanero-io/collections/blob/master/ci/tekton/README.md) which describes the steps needed to build your collections and deploy an NGINX server to host them into your Kabanero instance.
```

Comment:
Only slight mods made to this when I brought it in to the guide, which may or may not want to include:

-Github *E*nterprise
- artefacts/artifacts (yes, U.S spelling)
- ... README.md)*,* which describes.......

Reply:
I updated to resolve these.